### DISCUSSION TITLE
Version 1.0 request

### DISCUSSION THREAD

**Discussion by MarinusK** [Andy-Big/MKS-DLP-alter#7] [General]
What would be needed to get this working with v1.0? I am having trouble with the board at the moment, got everything working, custom GUI and all but now the board "freezes" while printing, elapsed time still counts but no triggering of the led or movement.

**Comment by Andy-Big** [comment]
Sorry for the long silence, I have been very busy lately :(
To adapt the firmware to version 1.0 of the board, I need to have such a board. The circuits of these boards differ from each other. And I can theoretically change the firmware for board 1.0, but without checking, I cannot guarantee that everything will work and nothing will fail :(
